packages/angular/.storybook/preview.ts

```diff
@@ -0,0 +1,46 @@
+import type { Preview } from '@storybook/angular'
+import { themes } from '@storybook/theming'
+
+// import { setCompodocJson } from '@storybook/addon-docs/angular'
+// import docJson from '../../../demo/documentation.json'
```

Comment:
do we need these?

Reply:
Just took them out. It's part of Storybook for generating docs but I don't really see any benefit.